template/report.html.tmpl

```diff
@@ -348,5 +349,9 @@
         <pre>{{ $dat.JSON }}</pre>
       </details>
     {{ end }}
+
+    <hr>
+    <h3>First Request to Relay</h3>
+    <pre>{{ .RelayMetrics.first_request }}</pre>
```

Comment:
Noticed this is an effect of `map[string]interface{}`, we end up accessing subfields as they were serialized to JSON, and not as we call them in the Go struct defined in the loadgen.